.changeset/yellow-ligers-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/xai': patch
```

Comment:
should this be a minor since they're new models?

Reply:
no, we use patch even for adding new models

Reply:
we don't follow semver - see https://sdk.vercel.ai/docs/migration-guides/versioning